engine/jobmaster/dm/dm_jobmaster.go

```diff
@@ -200,12 +200,7 @@ func (jm *JobMaster) OnJobManagerMessage(topic p2p.Topic, message interface{}) e
 // OnWorkerMessage implements JobMasterImpl.OnWorkerMessage
 func (jm *JobMaster) OnWorkerMessage(worker lib.WorkerHandle, topic p2p.Topic, message interface{}) error {
 	log.L().Debug("on worker message", zap.String("id", jm.workerID), zap.String("worker_id", worker.ID()))
-	// TODO: handle DDL request
-	response, ok := message.(dmpkg.MessageWithID)
-	if !ok {
-		return errors.Errorf("unexpected message type %T", message)
-	}
-	return jm.messageAgent.OnWorkerMessage(response)
+	return nil
```

Comment:
Seems framework's message interface is no longer useful?

Reply:
Useless now.

Reply:
Will `OnWorkerMessage` be removed? @liuzix